packages/block-editor/src/components/rich-text/multiline.js

```diff
@@ -15,6 +16,115 @@ import { store as blockEditorStore } from '../../store';
 import { useBlockEditContext } from '../block-edit';
 import { getMultilineTag } from './utils';
 
+function useEnterRef( props ) {
```

Comment:
Nothing in this whole file has change, except moving from an `onKeyDown` React listener to a native DOM one, so that we run it before the default rich text Enter handler.

Reply:
It's worth noting that we were actually not using the window listener until a couple of months ago, so it reverts some logic of #54543.